keyboards/moonlander/moonlander.c

```diff
@@ -358,6 +364,16 @@ void keyboard_post_init_kb(void) {
 
 #if defined(AUDIO_ENABLE) && defined(MUSIC_MAP)
 // clang-format off
+#ifdef HALFMOON
+const uint8_t music_map[MATRIX_ROWS][MATRIX_COLS] = LAYOUT_halfmoon(
+    58, 59, 60, 61, 62, 63, 64,
+    44, 45, 46, 47, 48, 49, 50,
+    30, 31, 32, 33, 34, 35, 36,
+    18, 19, 20, 21, 22, 23,
+     8,  9, 10, 11, 12,      3,
+                     0,  1,  2
+);
+#else
```

Comment:
Since the number of columns isn't changing, we don't need this change. Effectively, it should be the same.  In fact, we probably want this: 

```suggestion
#ifdef HALFMOON
const uint8_t music_map[MATRIX_ROWS][MATRIX_COLS] = LAYOUT_halfmoon(
    29, 30, 31, 32, 33, 34, 35,
    22, 23, 24, 25, 26, 27, 28,
    15, 16, 17, 18, 19, 20, 21,
     9, 10, 11, 12, 13, 14,
     4,  5,  6,  7,  8,      3,
                     0,  1,  2
);
#else
```